misc/python/materialize/output_consistency/input_data/params/enum_constant_operation_params.py

```diff
@@ -55,3 +55,7 @@
 REPETITIONS_PARAM = EnumConstantOperationParam(
     {"0", "1", "2", "10", "100", "-2"}, add_quotes=False
 )
+
+PRECISION_PARAM = EnumConstantOperationParam(
```

Comment:
precisions of 6 and 9 would be more common than 5 and 8 I think.

Reply:
✅ Adjusted.